.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build_and_test:
     docker:
-      - image: circleci/node:6.10
+      - image: circleci/node:8.0
```

Comment:
Should we all be using node 8.0? Should we update the README to reflect that?

Reply:
This was to simplify the unit testing process - since the un-built code uses async / await. Going to see if I can work around that so we don't have to upgrade node.